Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -209,8 +209,42 @@ bool EventRegionContext::shouldConsolidateInteractionRegion(IntRect bounds, Rend
     return false;
 }
 
+void EventRegionContext::addOcclusionMarginsForSmallInteractionRegions()
+{
+    constexpr int minimumSize = 20;
+    constexpr int occlusionMargin = 10;
+    Vector<InteractionRegion> interactionRegionsWithOcclusions;
+    for (auto& interactionRegion : m_interactionRegions) {
```

Comment:
we should only look at `Type::Interaction` here, no need to shield the existing occlusions.
(I get that we still need to append Occlusions to the final vector but I have another comment below :))